### PR TITLE
NTRN-357 Oracle docs

### PR DESCRIPTION
**Pull request by quasisamurai**

### CODE REVIEW

File: docs/neutron/token-generation-event/oracle/queries.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+This contract accepts these query msgs:
+
+```rust
+pub enum QueryMsg {
+    /// Multiplies a token amount (token that's present in the target pool for the TWAP) by the latest TWAP value for that token.
+    Consult {}, // returns `AssetInfo, Uint256`
+
+    /// Returns token TWAP value for given height.
+    TWAPAtHeight {}, // returns `AssetInfo, Deciimal256`
```
Comment (foxpy) [inline docs/neutron/token-generation-event/oracle/queries.md:11]: What is a "given" height? Querier doesn't provide any height in this context, so a height cannot be "given". I guess you wanted to write something like "last height for which a TWAP is recorded"? Or am I wrong?

**Review by foxpy — COMMENTED** [review]
Is `docs/neutron/token-generation-event/overview.md` supposed to be empty?

**Review by oopcode — CHANGES_REQUESTED** [review]
Did you try running the docs server locally? You didn't add this file to sidebar.js, so I'm not sure it's possible to actually see it in the UI.

File: docs/neutron/token-generation-event/oracle/queries.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Queries
+
+This contract accepts these query msgs:
```
Comment (oopcode) [inline docs/neutron/token-generation-event/oracle/queries.md:3]:
```suggestion
This contract accepts the following query msgs:
```

File: docs/neutron/token-generation-event/oracle/overview.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Oracle
+NOTE: this contract is an [Astroport Oracle](https://docs.astroport.fi/docs/develop/smart-contracts/oracle) which is modified to store historical TWAP values.
+## Overviewew
```
Comment (oopcode) [inline docs/neutron/token-generation-event/oracle/overview.md:3]:
```suggestion
## Overview
```

File: docs/neutron/token-generation-event/oracle/overview.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Oracle
```
Comment (oopcode) [inline docs/neutron/token-generation-event/oracle/overview.md:1]: I don't see a section that describes the diff from the original twap contract, please add it

**Review by oopcode — CHANGES_REQUESTED** [review]

File: docs/neutron/token-generation-event/oracle/queries.md (hunk 1 above)
Comment (quasisamurai) [reply docs/neutron/token-generation-event/oracle/queries.md:11]: 1. It is. It is placeholder for now. 
2. TWAPAtHeight here missing some params, there are actually 2 of them

**Review by oopcode — APPROVED** [review]

**Pull request merged**